src/Form/Form.js

```diff
@@ -77,30 +76,6 @@ class Form {
             }
         })
 
-        this.mutObsCount = 0
-        this.mutObsConfig = { childList: true, subtree: true }
-        this.mutObs = new MutationObserver(
-            (records) => {
-                const anythingRemoved = records.some(record => record.removedNodes.length > 0)
-                if (anythingRemoved) {
-                    // Must check for inputs because a parent may be removed and not show up in record.removedNodes
-                    if ([...this.inputs.all].some(input => !input.isConnected)) {
-                        // If any known input has been removed from the DOM, reanalyze the whole form
-                        window.requestIdleCallback(() => {
-                            this.formAnalyzer = new FormAnalyzer(this.form, input, this.matching)
-                            this.recategorizeAllInputs()
-                        })
-
-                        this.mutObsCount++
-                        // If the form mutates too much, disconnect to avoid performance issues
-                        if (this.mutObsCount >= MAX_FORM_MUT_OBS_COUNT) {
-                            this.mutObs.disconnect()
-                        }
-                    }
-                }
-            }
-        )
-
```

Comment:
This wasn't directly related to the breakage, but I'm dropping it now before investigating things further in the next few days.